### PR TITLE
Users can upload or change their Profile pixture or Cover photo

### PR DESCRIPTION
**Pull request by goodnessaig1**

### CODE REVIEW

File: src/Auth/Actions/userActions.js [hunk 1]
```diff
@@ -186,4 +186,61 @@ export const LogoutUser = (history) =>{
     
     history.push("/sign_in");
   };
-};
\ No newline at end of file
+};
+
+
+export const UploadProfilePIx = (formData, history) =>{
```
Comment (courage173) [inline src/Auth/Actions/userActions.js]: what is PIx??

File: src/Auth/Actions/userActions.js [hunk 2]
```diff
@@ -186,4 +186,61 @@ export const LogoutUser = (history) =>{
...
+        {
+            headers: {
+                
+                "Content-Type": "multipart/form-data",
+                'Accept': 'multipart/form-data',
+            },
+        });
+         promise.then(
+            function (payload) {
+                const  {data}  = payload;
```
Comment (courage173) [inline src/Auth/Actions/userActions.js:221]: i don't understand this if else to check error. is the error not suppose to come in a catch block, and where is the catch block

**Review by courage173 — CHANGES_REQUESTED** [review]
Please the UI issues and the additional issues here. thanks

File: src/Auth/Actions/userActions.js (hunk 1 above)
Comment (goodnessaig1) [reply src/Auth/Actions/userActions.js]: I have changed it to photo now sit

File: src/Auth/Actions/userActions.js (hunk 2 above)
Comment (courage173) [reply src/Auth/Actions/userActions.js:221]: goodness please update all your actions. for example uploading can be like this
```
export const UploadCoverPhoto = (formData, setCoverImg, setSubmitting) =>{
    return async (dispatch) => {
            const promise =   apiRequest('PATCH', `auth/v1/cover_photo`,formData,
                {
                    headers: {
                        "Content-Type": "multipart/form-data",
                        'Accept': 'multipart/form-data',
                    },
                });
                promise.then(payload => {
                        const  {data}  = payload; 
                            toast.success('Successful', {position: toast.POSITION.TOP_RIGHT});
                            dispatch(changePictureSuccess(data))
                    }).catch((error) => {
                      toast.error('An Error occured, Please try again', {position: toast.POSITION.TOP_RIGHT});
                     })
                  
                return promise;
    }
}
```

**Review by courage173 — CHANGES_REQUESTED** [review]

File: src/Components/Profile/UserCoverModal.js [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+                <hr className='upload_hr'/>
+            </div>
+            <Formik
+                initialValues={{
+                    photo: ''
+                }}
+                onSubmit={(values, {setSubmitting,}) => {
+                    let formData = new FormData();
+                    formData.append(`photo`, values.photo);
+                    UploadCoverPhoto(formData, setSubmitting).then(response => {
```
Comment (courage173) [inline src/Components/Profile/UserCoverModal.js]: now think about this, you don't need to pass setSubmitting you can do it inside the .then

**Review by courage173 — CHANGES_REQUESTED** [review]

**Pull request merged**